### DISCUSSION TITLE
Mermaid doesn't work in gists

### DISCUSSION THREAD

**Discussion by donno2048** [community/community#11506] [Other Feature Feedback, Questions, & Ideas] [answered]
https://github.blog/2022-02-14-include-diagrams-markdown-files-mermaid/

**Comment by davidstaheli** [comment]
@donno2048, thanks for raising this. It's definitely important to have Mermaid support in gists. We wanted to quickly release initial, basic support for Mermaid, and that ended up being without gist support. But it's high on our list of items to support soon. This discussion will be good for tracking it.

**Reply by euthuppan** [reply]
@davidstaheli I don't think it's working in wiki pages either. Seems to work fine in a repo's README.md though.

**Comment by davidstaheli** [comment]
Hi all. Mermaid support in gists should be released very soon, perhaps within one or two weeks. Thanks for your patience with this.

**Comment by davidstaheli** [comment, marked as answer]
Gist support for Mermaid is now shipped and in production on github.com -- sooner than expected!

**Reply by davilima6** [reply]
hi @davidstaheli . still in github? would u be able to check if there was a regression? Mermaid charts are rendered after save, but not in Preview.

**Comment by RobertHana** [comment]
Hmmm, mermaid diagrams are not rendering in my "All Gists" tab.

**Comment by tenderlove** [comment]
Mermaid renders in previews on Gists, but not after you comment.

I uploaded a video here:

https://github.com/community/community/assets/3124/f1f4d076-e569-4f0b-ab3d-4fe3b997d9cb

**Comment by yoponchik** [comment]
Same here, not working...

**Comment by rusackas** [comment]
Sorry to resurrect an ancient thread, but it seems mermaid is still not working in gists... but it does still (as expected) work on repo `.md`  files and on wikis. 

```mermaid
flowchart TD
  MM[Mermaid Support] -- yes --> WIKI[Wikis];
  MM -- yes --> Issues[Issue/PR threads]
  MM -- yes --> MD[Markdown Files]
  MM -- no --> Gist[Gists]
 ```

**Comment by cristiano-belloni** [comment]
Not working for me too.

**Comment by adpthegreat** [comment]
same , it says failed to load preview
